### ISSUE TITLE
Magentacloud (German Telekom cloud) login broken due to tags-eu.tiqcdn.com

### ISSUE THREAD

**Issue by marcusfey** [StevenBlack/hosts#2442]
Hi there,

Deutsche Telekom (probably biggest ISP in Germany) hosts a cloud service under magentacloud.de. They changed the front-end recently and now reference tags-eu.tiqcdn.com which is blocked [here](https://github.com/StevenBlack/hosts/blame/1f745934590cc289fda9cd899a420b4d3ef2af30/data/adaway.org/hosts#L10349). This breaks the website and stops the login from being loaded/generated.

I am aware of #2189 regarding tags-eu.tiqcdn.com. However the line is two years old and yoyo.org shows the same date/year.

Would it be possible/sensible for you to remove the line or should I get back to yoyo.org?

Thanks!

**Comment by StevenBlack** [comment]
Thank you for this Marcus @marcusfey.

I just refreshed our cache from yoyo.org.  We serve yoyo.org from a private cache because I am still not convinced about blocking twitter's `t.co` domain. But my views on `t.co` is gradually changing, a little bit each day.

The domain `tags-eu.tiqcdn.com` is still blocked by yoyo.org and I agree with that.

Magentacloud has coded their site so it's critically dependent on `tags-eu.tiqcdn.com` and their site is broken if `tags-eu.tiqcdn.com` does not respond.  This is, therefore, Magentacloud's bug because the are doing this by choice.

We don't serve Magentacloud here. We serve people who don't want to be tracked or served malware and, optionally, other things like porn, etc.

Thank you for this report but this isn't going to change at this time.

Closing.
